open-vm-tools-centos/config.json.template

```diff
@@ -241,23 +399,21 @@
                 "rprivate"
             ]
         },
-
-        {
-            "destination": "/proc",
-            "type": "proc",
-            "source": "proc"
-        },
         {
+            "destination": "/dev/rtc",
             "type": "bind",
-            "source": "/dev",
-            "destination": "/dev",
+            "source": "/dev/rtc",
             "options": [
-                "rbind",
                 "rw",
-                "mode=755"
+                "rbind"
             ]
         },
         {
+            "destination": "/proc",
```

Comment:
It seems that /hostproc and /proc is bound earlier in the file, is there a reason to do this (with no options)?